### PR TITLE
Fix/page typespecs

### PR DESCRIPTION
**Pull request by r-coh**
Fixes none

## Proposed Changes

  - add doc for page controller plug
  - add type specification for page controller plug
  -

### CODE REVIEW

File: lib/spender_web/controllers/page_controller.ex [hunk 1]
```diff
@@ -8,7 +8,7 @@ defmodule SpenderWeb.PageController do
   it then convertes this to content that can be displayed on a web page and stores it in
   a Connection struct and returns the struct.
   """
-  @spec index(Plug.Conn.t(), any())::Plug.Conn,t()
+  @spec index(Plug.Conn.t(), any())::Plug.Conn.t()
```
Comment (zacck-zz) [inline lib/spender_web/controllers/page_controller.ex]: @br4inii  This is good work! However, this needs to be formatted better please run `mix format lib/spender_web/controllers/page_controller.ex`

File: lib/spender_web/controllers/page_controller.ex [hunk 2]
```diff
@@ -3,9 +3,12 @@ defmodule SpenderWeb.PageController do
   Controller to handle display of pages when the API is accessed from the browser
   """
   use SpenderWeb, :controller
-
-  # this function needs an @doc attribute
-  # this function needs a spec
+  @doc """
+  This function takes a Connection Struct and any other parameter which may be optional
```
Comment (zacck-zz) [inline lib/spender_web/controllers/page_controller.ex]: I think this is a slightly better description of the function
```
This function takes a Connection Struct and an optional parameter
it then converts it then renders the landing page for the root route of the Mula API
```

**Review by zacck-zz — CHANGES_REQUESTED** [review]
Hey I left some changes for you

**Review by zacck-zz — APPROVED** [review]
@br4inii  thank you for your work

**Comment by r-coh** [comment]
@zacck you welcome

**Pull request closed without merging**